pkg/datastore/helper.go

```diff
@@ -66,33 +66,35 @@ func FromEnv(l *logrus.Entry, authorizer authorizer.Authorizer, instancer author
 		SSL_MODE_ENV_VAR,
 	} {
 		if _, isPresent := os.LookupEnv(envVar); !isPresent {
-			err = ErrMissingEnvVar.WithValue(ENV_VAR, envVar)
-			l.Error(err)
-			return nil, err
+			panic(ErrMissingEnvVar.WithValue(ENV_VAR, envVar))
```

Comment:
I understand why it makes sense to panic when a required env. variable is missing, but I think that this function, which just generates a struct `DBConfig`, should additionally return `error` instead of panicking. A consumer (`FromEnv` in this library) should then panic/exit if it got back an error. Maybe we will have a consumer who wants to try to generate `DBConfig` based on env. variables, and if it's not possible, instantiate `DBConfig` manually with hardcoded values?\
But if you wanna panic in this function, it's important that the header comments mention under which circumstances this function will panic.

Reply:
done